### ISSUE TITLE
Build failed at master

### ISSUE THREAD

**Issue by Litrop** [second-state/SSVM#157]
I follow [build-ssvm](https://github.com/second-state/SSVM#build-ssvm) to build at master branch in this command:
```sh
docker run --rm -w /ssvm secondstate/ssvm bash -c \
    "git clone --depth 1 https://github.com/second-state/SSVM.git . && mkdir -p build && cd build && cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=ON .. && make -j"
```
But it fails with error, here is log:
```log
[ 81%] Building CXX object lib/vm/CMakeFiles/ssvmVM.dir/vm.cpp.o
[ 82%] Building CXX object tools/ssvm/CMakeFiles/ssvmc.dir/ssvmc.cpp.o
[ 83%] Building CXX object test/aot/CMakeFiles/ssvmAOTWagonTests.dir/AOTwagonTest.cpp.o
[ 84%] Linking CXX executable ssvmLoaderEthereumTests
[ 85%] Linking CXX executable ssvmASTTests
In file included from /ssvm/tools/ssvm/ssvmc.cpp:5:
/ssvm/build/include/common/version.h:22:5: error: use of undeclared identifier 'CPACK_PACKAGE_VERSION'
    CPACK_PACKAGE_VERSION;
    ^
1 error generated.
make[2]: *** [tools/ssvm/CMakeFiles/ssvmc.dir/build.make:63: tools/ssvm/CMakeFiles/ssvmc.dir/ssvmc.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2373: tools/ssvm/CMakeFiles/ssvmc.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 86%] Linking CXX executable ssvmLoaderWagonTests
[ 87%] Linking CXX executable ssvmProcessTests
[ 88%] Linking CXX executable wasiTests
[ 88%] Built target ssvmLoaderFileMgrTests
[ 89%] Linking CXX executable ssvmAOTWagonTests
[ 90%] Linking CXX static library libssvmVM.a
[ 90%] Built target ssvmLoaderEthereumTests
[ 90%] Built target ssvmVM
[ 90%] Built target ssvmLoaderWagonTests
[ 90%] Built target ssvmProcessTests
[ 90%] Built target ssvmASTTests
[ 90%] Built target wasiTests
[ 90%] Built target ssvmAOTWagonTests
make: *** [Makefile:163: all] Error 2
```
Is there anything wrong with the command?

**Comment by hydai** [comment]
Hi, thanks for your question.

We use CMake command to generate version information from the git tags. Because you only fetch the latest commit which doesn't contain the `git tags` information. That's why you got this error.

Could you please remove the `--depth` or fetch at least one tag?

**Comment by Litrop** [comment]
Thank for your reply.
Build success without `--depth 1`! But it's confused to build with git tags information.
Close for solving my issue.
Thanks.

**Comment by hydai** [comment]
Got it. We will figure out a new way to generate version number to avoid this issue. Thanks!

**Comment by hydai** [comment]
This is fixed at #160
